user/repository-providers.md

```diff
@@ -4,8 +4,8 @@ layout: en
 
 ---
 
-Build and test your open source projects hosted on GitHub on [travis-ci.org](https://travis-ci.org/).
+<!-- FIXME: do we need this? -->
 
-Build and test your private repositories hosted on GitHub on [travis-ci.com](https://travis-ci.com/).
+Build and test your open source and private repositories hosted on GitHub on [travis-ci.com](https://travis-ci.com/).
```

Comment:
I think this normally helps with answering the question of which repository providers we support, clarifying what we support and what we don't. 

I think going forward, we can remove the full page, since it's being redirected to the "customizing the build section" https://github.com/travis-ci/docs-travis-ci-com/blob/master/user/customizing-the-build.md 

At: customizing-the-build/#What-repository-providers-or-version-control-systems-can-I-use%3F

Reply:
I've attempted to fix this in the commits below - let me know what you think!